ci/build.sh

```diff
@@ -71,65 +81,57 @@ if [ "$TRAVIS_BRANCH" != "master" ] && [ "$TRAVIS_EVENT_TYPE" == "push" ] && [ -
-    fluent_bit_end=$(($fluent_bit_end - 1))
-    
-    echo "Copy 'values.yaml' from line $fluent_bit_start to line $fluent_bit_end to 'fluent-bit-overrides.yaml'"
-    echo "# This file is auto-generated." > deploy/helm/fluent-bit-overrides.yaml
-    # Copy lines of fluent-bit section and remove indention from values.yaml
-    sed -n "$fluent_bit_start,${fluent_bit_end}p" deploy/helm/sumologic/values.yaml | sed 's/  //' >> deploy/helm/fluent-bit-overrides.yaml
-    # Remove release name from service name
-    sed -i 's/collection-sumologic/fluentd/' deploy/helm/fluent-bit-overrides.yaml
-
-    prometheus_start=`grep -n "prometheus-operator:" deploy/helm/sumologic/values.yaml | head -n 1 | cut -d: -f1`
-    prometheus_start=$(($prometheus_start + 2))
-    prometheus_end=`grep -n "## Configure falco" deploy/helm/sumologic/values.yaml | head -n 1 | cut -d: -f1`
-    prometheus_end=$(($prometheus_end - 1))
-    
-    echo "Copy 'values.yaml' from line $prometheus_start to line $prometheus_end to 'prometheus-overrides.yaml'"
-    echo "# This file is auto-generated." > deploy/helm/prometheus-overrides.yaml
-    # Copy lines of prometheus-operator section and remove indention from values.yaml
-    sed -n "$prometheus_start,${prometheus_end}p" deploy/helm/sumologic/values.yaml | sed 's/  //' >> deploy/helm/prometheus-overrides.yaml
-    # Remove release name from service name
-    sed -i 's/collection-sumologic/fluentd/' deploy/helm/prometheus-overrides.yaml
-
-    falco_start=`grep -n "falco:" deploy/helm/sumologic/values.yaml | head -n 1 | cut -d: -f1`
-    falco_start=$(($falco_start + 2))
-    falco_end=`grep -n "## Configure falco" deploy/helm/sumologic/values.yaml | head -n 1 | cut -d: -f1`
-    falco_end=$(($falco_end - 1))
-    
-    echo "Copy 'values.yaml' from line $falco_start to end to 'falco-overrides.yaml'"
-    echo "# This file is auto-generated." > deploy/helm/falco-overrides.yaml
-    # Copy lines of falco section and remove indention from values.yaml
-    sed -n "$falco_start,$ p" deploy/helm/sumologic/values.yaml | sed 's/  //' >> deploy/helm/falco-overrides.yaml
-
-    if [ "$(git diff deploy/helm/fluent-bit-overrides.yaml)" ] || [ "$(git diff deploy/helm/prometheus-overrides.yaml)" ] || [ "$(git diff deploy/helm/falco-overrides.yaml)" ]; then
-      echo "Detected changes in 'fluent-bit-overrides.yaml', 'prometheus-overrides.yaml' or 'falco-overrides.yaml', committing the updated version to $TRAVIS_BRANCH..."
-      git fetch origin-repo
-      git checkout $TRAVIS_BRANCH
-      git add deploy/helm/*-overrides.yaml
-      git commit -m "Generate new overrides yaml file(s)."
-      git push --quiet origin-repo "$TRAVIS_BRANCH"
-    else
-      echo "No changes in the generated overrides files."
-    fi
+  # Generate override yaml files for chart dependencies to determine if changes are made to overrides yaml files
+  echo "Generating overrides files..."
```

Comment:
Do we always generate override files now?

Reply:
We do.  We will only commit them if there is a difference detected, however.

Reply:
gtk